### PR TITLE
 Merge branch 'release/2.2' into 'master'

### PR DESCRIPTION
**Pull request by ajaybhargavb**

### CODE REVIEW

**Review by natemcmaster — APPROVED** [review]

**Review by mikaelm12 — APPROVED** [review]

**Review by JunTaoLuo — APPROVED** [review]

**Comment by mikaelm12** [comment]
Taking a look

File: src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs [hunk 1]
```diff
@@ -60,7 +60,11 @@ public async Task RunClientTests()
 
         private static async Task<SamplesDeploymentResult> CreateDeployments(ILoggerFactory loggerFactory)
         {
-            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("CORS");
+            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("Middleware");
+
+            var runtimeFlavor = GetRuntimeFlavor();
```
Comment (mikaelm12) [inline src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs]: Hmm, these are the changes that are failing. It looks like the method doesn't exist. These two lines weren't part of my change.  Any idea why this is failing @natemcmaster? 
```
error CS0103: The name 'GetRuntimeFlavor' does not exist in the current context [D:\a\1\s\src\Middleware\CORS\test\FunctionalTests\FunctionalTests.csproj]
```

Comment (mikaelm12) [reply src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs]: Thanks for your help!

File: src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs [hunk 2]
```diff
@@ -60,7 +60,9 @@ public async Task RunClientTests()
 
         private static async Task<SamplesDeploymentResult> CreateDeployments(ILoggerFactory loggerFactory)
         {
-            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("CORS");
+            var solutionPath = TestPathUtilities.GetSolutionRootDirectory("Middleware");
+
+
```
Comment (mikaelm12) [inline src/Middleware/CORS/test/FunctionalTests/CorsMiddlewareFunctionalTest.cs]:
```suggestion
```

**Pull request merged**